aemcmc/conjugates.py

```diff
@@ -0,0 +1,57 @@
+import aesara.tensor as at
+from etuples import etuple, etuplize
+from kanren import eq, lall
+from unification import var
+
+
+def beta_binomial_conjugateo(model_expr, observation_expr, posterior_expr):
```

Comment:
The underlying relation isn't clear from these arguments.  For example, why is there an `observation_expr`?  It seems like this should be implementing an equality with only two terms.